src/rpc/dial.rs

```diff
@@ -965,6 +1005,7 @@ async fn maybe_connect_via_webrtc(
                     let mut signaling_client = SignalingServiceClient::new(channel.clone());
                     match ice_candidate {
                         Some(ice_candidate) => {
+                            log::debug!("Gathered local candidate of {}", ice_candidate);
```

Comment:
(minor, opt) we aren't super consistent about it, but in general I think `"Gathered local candidate of {ice_candidate}"` is cleaner than `"Gathered local candidate of {}", ice_candidate`. Similar comment with `log` statement below. No need to change if you disagree!

Reply:
Yep can do! I always forget you can put the variable directly in the braces.